packages/git/src/browser/git-contribution.ts

```diff
@@ -148,7 +148,8 @@ export namespace GIT_COMMANDS {
     export const OPEN_CHANGES: Command = {
         id: 'git.open.changes',
         category: 'Git',
-        label: 'Open Changes'
+        label: 'Open Changes',
+        iconClass: 'theia-open-change-icon'
```

Comment:
here is the original changes: https://github.com/theia-ide/theia/commit/2519e5323cc6bdaa2bc4bccaabc6abf83dd81aaf#diff-23f6de2d4f77ec14174dd983bef7b2cc

please apply them, changes to `Open File` command should be recovered as well